### PR TITLE
001-introduction-to-bash [es]

### PR DESCRIPTION
**Pull request by marcelozarate**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Translation of "Introduction to Bash" to spanish.

## Related Tickets & Documents

000 - About the Author #68

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

File: ebook/es/content/001-introduction-to-bash.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Introducción al scripting de Bash
+
+¡Bienvenido a esta guía de entrenamiento de los fundamentos de Bash! En este **curso intensivo de Bash**, aprenderá los **aspectos básicos de Bash** para que pueda empezar a escribir sus propios scripts de Bash y automatizar sus tareas diarias.
+
+Bash es un lenguaje de comandos y consola de Unix. Está ampliamente disponible en varios sistemas operativos, y también es el intérprete de comandos por defecto en la mayoría de los sistemas Linux.
+
+Bash significa _Bourne-Again SHell_. Al igual que otros shells, puede usar Bash de forma interactiva directamente en tu terminal, y también, puede usar Bash como cualquier otro lenguaje de programación para escribir scripts. Este libro le ayudará a aprender los fundamentos de la escritura de scripts en Bash, incluyendo las variables de Bash, entrada del usuario, comentarios, argumentos, arreglos, expresiones condicionales, condicionales, bucles, funciones, depuración y pruebas.
+
+Para escribir scripts Bash, solo necesita un terminal UNIX y un editor de texto como Sublime Text, VS Code, o un editor basado en terminal como vim o nano.
```
Comment (marcelozarate) [inline ebook/es/content/001-introduction-to-bash.md:9]:
```suggestion
Para escribir scripts Bash, solo necesita un terminal UNIX y un editor de texto como Sublime Text, VS Code, o un editor basado en terminal como vim o nano.

```

Comment (marcelozarate) [reply ebook/es/content/001-introduction-to-bash.md:9]: Blank line at the end

**Review by marcelozarate — COMMENTED** [review]
Missing blank line at the end

**Review by bobbyiliev — APPROVED** [review]
Thank you! looks great!

**Pull request merged**